### PR TITLE
Alkwa/update js storybook

### PR DESCRIPTION
**Pull request by alkwa-msft**
# What
Wanted to take a better stab at describing how to use our JS bundles on a webpage. Also updated documentation on using the <meta> tag.

# Why
The bundles use mini-fication that requires utf-8  characters. If you do not expect these characters, then the bundle will fail to load.

# How Tested
Ran locally

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 1]
```diff
@@ -6,81 +6,142 @@ import { Meta } from '@storybook/addon-docs';
...
+    token: 'ACCCESS_TOKEN' // string
   },
-  elementToRender, // container element
+  containerElement, // container element
   props // Optional, Object contains all optional props for composite, check composite page for more details
 );
 ```
 
-Check [Chat Composite Basic Example](./?path=/docs/composites-chat-basicexample--basic-example) and [Call Composite Basic Example](./?path=/docs/composites-call-basicexample--basic-example) to see optional props of composites
+_[Call Composite Basic Example](./?path=/docs/composites-call-basicexample--basic-example) for details on what additional properties you can apply to the call composite_
```
Comment (prprabhu-ms) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:89]:
```suggestion
Check [Call Composite Basic Example](./?path=/docs/composites-call-basicexample--basic-example) for details on what additional properties you can apply to the call composite_
```

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 2]
```diff
@@ -6,81 +6,142 @@ import { Meta } from '@storybook/addon-docs';
...
+      userId: 'USERID', // string
+      token: 'ACCCESS_TOKEN' // string
+    },
+    document.getElementById('call-container'),
+    { key: new Date() }
+  );
+</script>
+```
+
+If you want to see these examples in Github [Examples](https://github.com/Azure/communication-ui-library/blob/PublicPreview/1.0.0-beta.3/samples/StaticHtmlComposites/index.html)
```
Comment (prprabhu-ms) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:140]: Weirdly worded sentence. Perhaps: 

```markdown
These examples are also available on [GitHub](...)
```

Also, why do we link to `PublicPreview/1.0.0-beta.3`. Let's update that to https://github.com/Azure/communication-ui-library/blob/1.3.0/samples/StaticHtmlComposites/index.html

But really, we should probably add a new tag `npm-latest` that always tracks the latest stable release and bump that each time we do a stable release.
Then this documentation can link to that.

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 3]
```diff
@@ -6,81 +6,142 @@ import { Meta } from '@storybook/addon-docs';
...
+
+If you are getting a syntax error at runtime. Don't forget to add the **`<meta charset="utf-8">`** tag in the header of your html code.
```
Comment (prprabhu-ms) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:142]:
```suggestion
❗ If you are getting a syntax error at runtime. Don't forget to add the **`<meta charset="utf-8">`** tag in the header of your html code.
```

**Review by prprabhu-ms — APPROVED** [review]

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 4]
```diff
@@ -1,86 +1,150 @@
...
   {
-    displayName, // string
-    threadId, // string
-    endpoint, // string
-    userId, // Object { communicationUserId: string }
-    token // string
+    displayName: 'DISPLAY_NAME', // string
+    threadId: 'THREAD_ID', // string
+    endpoint: 'ENDPOINT', // string
+    userId: 'USERID', //
```
Comment (PorterNan) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:59]: Did we change this to string?

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 5]
```diff
@@ -1,86 +1,150 @@
...
   //The for composite which allows you control composite externally,
   // This function will be exposed to window.callComposite once you import composite bundle
   {
-    groupId, // string, Provide any GUID to join a group
-    displayName, // string
-    userId, // Object { communicationUserId: string }
-    token // string
+    displayName: 'DISPLAY_NAME', // string
+    groupId: 'GROUPID', // string
+    userId: 'USERID', // string
```
Comment (PorterNan) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:81]: Same here, 
userId: {communicationUserId: 'UserID'}

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 6]
```diff
@@ -1,86 +1,150 @@
...
+<!-- Replace this url with your own CDN in production environment -->
+<script src="https://github.com/Azure/communication-ui-library/releases/latest/download/callComposite.js"></script>
+<!-- height need to be set for composite to fit the layout -->
+<div id="call-container" style="height: 50vh"></div>
+<script type="module">
+  const callAdapter = await callComposite.loadCallComposite(
+    {
+      displayName: 'DISPLAY_NAME', // string
+      groupId: 'GROUPID', // string
+      userId: 'USERID', // string
```
Comment (PorterNan) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:131]: userId: {communicationUserId: 'USERID'}

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx [hunk 7]
```diff
@@ -1,86 +1,150 @@
...
     {
-      displayName: displayName,
-      threadId: threadId,
-      endpoint: endpoint,
-      userId: user,
-      token: token
+      displayName: 'DISPLAY_NAME', // string
+      threadId: 'THREAD_ID', // string
+      endpoint: 'ENDPOINT', // string
+      userId: 'USERID', //
```
Comment (PorterNan) [inline packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:109]: userId: {communicationUserId: 'USERID'}

**Review by PorterNan — APPROVED** [review]

**Comment by prprabhu-ms** [comment]
Bumping off my attention set.

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx (hunk 7 above)
Comment (alkwa-msft) [reply packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:109]: Changing to : { communicationUserId: 'USERID' } // CommunicationUserIdentifier

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx (hunk 6 above)
Comment (alkwa-msft) [reply packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:131]: Changing to : { communicationUserId: 'USERID' } // CommunicationUserIdentifier

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx (hunk 4 above)
Comment (alkwa-msft) [reply packages/storybook/stories/NonReact/NonReactComposites.stories.mdx:59]: changing this to : { communicationUserId: 'USERID' }, // CommunicationUserIdentifier

File: packages/storybook/stories/NonReact/NonReactComposites.stories.mdx (hunk 5 above)
Comment (JamesBurnside) [reply packages/storybook/stories/NonReact/NonReactComposites.stories.mdx]: updated

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**